src/test/java/hudson/plugins/active_directory/docker/ActiveDirectoryGenericContainer.java

```diff
@@ -35,8 +38,8 @@ public ActiveDirectoryGenericContainer() {
     public ActiveDirectoryGenericContainer<SELF> withStaticPorts() {
         addFixedExposedPort(3268, 3268); // global catalog
         addFixedExposedPort(3269, 3269); // global catalog over tls
-        addFixedExposedPort(53, 53, InternetProtocol.TCP); // DNS over TCP
-        addFixedExposedPort(53, 53, InternetProtocol.UDP); // DNS over UDP
+        addFixedExposedPort(553, 53, InternetProtocol.TCP); // DNS over TCP
+        addFixedExposedPort(553, 53, InternetProtocol.UDP); // DNS over UDP
```

Comment:
works around systemd (resolved) starting a listening DNS server  (why!!???)
https://www.linuxuprising.com/2020/07/ubuntu-how-to-free-up-port-53-used-by.html  whilst there was an attempt to disable this in the CI infra, it would have caused issues anyway for local devs.  so use a different port and configure a non standard port in the code